package.jsonのmain項目とdistの出力ファイルが食い違っています

いつもお世話になっています。

React/viteでOudOperatorを使用しています。  
viteで実行開始したところ  
  ✘ [ERROR] Failed to resolve entry for package "@route-builders/oud-operator". The package may have incorrect main/module/exports specified in its package.json. [plugin vite:dep-scan]  
というエラーが出ています。

package.jsonで「  "main": "./dist/O_O.js",」と設定されていますが、distフォルダにあるのはindex.jsなので、
齟齬が生じていると思います。
こちらの項目を「  "main": "./dist/index.js",」に変更、もしくはdist/index.jsをdist/O_O.jsに変更すると問題なく利用することができました。

O_O.jsかindex.jsのどちらかに同一していただくとありがたいです。

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@route-builders/oud-operator",
   "version": "3.0.0",
   "description": "The library to use the string read from OuDia file",
-  "main": "./dist/O_O.js",
+  "main": "./dist/index.js",
   "types": "./types/index.d.ts",
   "author": {
     "name": "up-tri",

